lib/ably/rest/channel.rb

```diff
@@ -58,6 +58,10 @@ def publish(name, data = nil, attributes = {})
         messages = if name.kind_of?(Enumerable)
           name
         else
+          if name.kind_of?(Ably::Models::Message)
```

Comment:
Maybe this suggestion breaks some requirement in the spec, but I'd be nice if the the lib automatically built an array when just one message is passed as an argument.

Reply:
Yeh, well I was thinking of doing that, but it's not defined in the spec elsewhere so wasn't sure I wanted to add a new overloaded method...